### ISSUE TITLE
Is anyone successful for configuring the environment?

### ISSUE THREAD

**Issue by ZiqiLoveSunshine** [ivalab/AffKpNet#5]
First of all, I tried python==3.6 and follow the step of configuring the environment of `maskrcnn-benchmark` as mentioned, when I succeded with the environment of `maskrcnn-benchmark`, I returned to this project and tried to install

```
cd AffKpNet 
python setup.py install
```
however, it told me that python version must>=3.8

so then,
I used python==3.8, torch==1.6.0, torchvision==0.7.0, cudatoolkit=10.1
and followed the steps of `maskrcnn-benchmark` as well. It compiles successful, However, when it comes to execute the code, there is always some errors
`subprocess.CalledProcessError: Command '['ninja', '-v']' returned non-zero exit status 1.`

how should I do? I have no idea.

**Comment by Knightbreeze** [comment]
@GuoyuloveSunshine  Hello, I recently are trying to configure the environment, met the same problem with you, you then have to solve the `subprocess. CalledProcessError: Command '['ninja', '-v']' returned non-zero exit status 1.` this problem?

**Comment by ZiqiLoveSunshine** [comment]
No, I give up configuring this project, there are lots of compiling errors that I cannot solve, I probably use the dataset that they provide to train another network later.

**Comment by Knightbreeze** [comment]
@GuoyuloveSunshine  Thanks for your reply. I failed after many attempts. May I ask if you have any new networks about the affordance detection and used for manipulator fetching? I want to learn and replicate related aspects recently, but there are few open source RGB-D image-based networks during the search on the Internet. So if there is a new network also hope to recommend some.

**Comment by ZiqiLoveSunshine** [comment]
I am sorry I cannot give you any useful advice about the network, as I begin to tacile with the manipulation trajectories and didn't insist on the grasping part, perheps you can try to search 'manipulation affordance' or 'grasping', There may be some repository that can help you.
